.github/workflows/bazel-cache-push.yml

```diff
@@ -65,6 +65,14 @@ jobs:
         run: |
           cd lte/gateway
           vagrant ssh -c 'cd ~/magma; bazel build //...' magma
+      - name: Test all with Bazel `bazel test //... --cache_test_results=no`
+        run: |
+          cd lte/gateway
+          vagrant ssh -c 'cd ~/magma; bazel test //... --cache_test_results=no' magma
```

Comment:
more a conceptual question: do we want to let the job fail if a bazel run fails here? the job is designed to build caches - the build and the tests are verified in another workflow.

Reply:
I think for now, failing is OK. I wouldn't have caught the issue on master if I didn't see everything red :D